.ci/test_r_package.sh

```diff
@@ -167,45 +167,43 @@ elif [[ $R_BUILD_TYPE == "cran" ]]; then
     if [[ "${TASK}" == "r-rchk" ]]; then
         echo "Checking R package with rchk"
         mkdir -p packages
-        cp ${PKG_TARBALL} packages
+        cp "${PKG_TARBALL}" packages
         RCHK_LOG_FILE="rchk-logs.txt"
         docker run \
-            -v $(pwd)/packages:/rchk/packages \
+            -v "$(pwd)"/packages:/rchk/packages \
             kalibera/rchk:latest \
             "/rchk/packages/${PKG_TARBALL}" \
-        2>&1 > ${RCHK_LOG_FILE} \
+         > ${RCHK_LOG_FILE} 2>&1 \
         || (cat ${RCHK_LOG_FILE} && exit 1)
         cat ${RCHK_LOG_FILE}
 
         # the exceptions below are from R itself and not LightGBM:
         # https://github.com/kalibera/rchk/issues/22#issuecomment-656036156
-        exit $(
+        exit "$(
```

Comment:
What error did `shellcheck` raise here? Did it perhaps actually want you to put quotes around `RCHK_LOG_FILE` on the line below, instead?

Reply:
In testing.sh line 182:
        exit $(
             ^-- SC2046 (warning): Quote this to prevent word splitting.

Reply:
hmmmm ok. I think I disagree with `shellcheck` on this one, and it's a bit strange to see a `"` at the beginning of such a multiline statement.

Can you please revert the quotes for this statement, and instead add a comment directly above (indented over to match the `exit`) like this?

```text
# shellcheck disable=SC2046
```